packages/client/src/v2-events/features/events/actions/correct/request/Review.tsx

```diff
@@ -97,25 +65,30 @@ export function Review() {
           actionConfig.label,
           previousFormValues
         )}
-        onEdit={handleEdit}
+        onEdit={({ pageId, fieldId }) => {
+          navigate(
+            ROUTES.V2.EVENTS.REQUEST_CORRECTION.PAGES.buildPath(
+              { pageId, eventId },
+              {
+                from: 'review'
+              },
+              fieldId ? makeFormFieldIdFormikCompatible(fieldId) : undefined
```

Comment:
(it's not your change but:) feels kinda lame that the 'hack' isn't contained in a one transform layer. Could we/have we thought about this?